### PR TITLE
Changes to display error when export HTML format results in some error

### PR DESCRIPTION
**Pull request by poleveD**
- export for HTML format uses 
- When incorrect theme name is provided, themeServer returns a statuscode of "200" and a response.text='Theme could not be found in the npm registry.'
- When theme name is correct, response.text contains the HTML to be rendered in the browser

The fix I added is more of a stop gap until the response from themeServer is corrected to return an error code/error message.

### CODE REVIEW

**Comment by poleveD** [comment]
@thomasdavis can you please merge this one?

**Comment by thomasdavis** [comment]
Just left one comment before merging.

**Comment by poleveD** [comment]
Modified code to show forward slash in the path for ```export``` command. Please check.

**Comment by justsml** [comment]
Hey guys - please check out my PR #103 - I used Node's path.resolve to solve this. I've only tested on Linux and Mac, but I think it will work on Windows too.
Let me know if you have any feedback.

File: lib/exportResume.js [hunk 1]
```diff
@@ -71,8 +66,14 @@ function sendExportRequest(resumeJson, fileName, theme, format, callback) {
         })
         .set('Accept', 'application/json')
         .end(function(response) {
-            fs.writeFileSync(process.cwd() + '/' + fileName + format, response.text);
-            callback();
+		    var errorText = null,
+			    responseText = response.text;
+		    if(responseText && responseText.indexOf('<html>') > -1) {
```
Comment (dbkaplun) [inline lib/exportResume.js]: This is a rather strange method of error handling. Can you check if `.statusCode === 200` instead?

**Pull request closed without merging**